eth/tools/builder/chain/builders.py

```diff
@@ -312,11 +319,18 @@ def disable_pow_check(chain_class: Type[BaseChain]) -> Type[BaseChain]:
     if not chain_class.vm_configuration:
         raise ValidationError("Chain class has no vm_configuration")
 
-    chain_class_without_seal_validation = type(
-        chain_class.__name__,
-        (NoChainSealValidationMixin, chain_class),
-        {},
-    )
+    if issubclass(chain_class, NoChainSealValidationMixin):
+        chain_class_without_seal_validation = type(
+            chain_class.__name__,
```

Comment:
Same here, instead of doing the additional subclass, lets just return `chain_class` unmodified.